src/os/bluestore/BlueStore.cc

```diff
@@ -11078,7 +11076,6 @@ int BlueStore::_omap_rmkey_range(TransContext *txc,
   {
     const string& prefix =
       o->onode.is_pgmeta_omap() ? PREFIX_PGMETA_OMAP : PREFIX_OMAP;
-    o->flush();
     get_omap_key(o->onode.nid, first, &key_first);
     get_omap_key(o->onode.nid, last, &key_last);
     txc->t->rm_range_keys(prefix, key_first, key_last);
```

Comment:
rm_range_keys impl enumerates keys, so we need the flush.  

we could perhaps have a KeyValueDB method to tell us whether a proper range delete is implemented and if so skip the flush, but we have that off by default still (rocksdb recommends not using yet iirc?).

Reply:
ok

Reply:
ok